corehq/ex-submodules/casexml/apps/phone/restore.py

```diff
@@ -218,7 +303,20 @@ def get_case_payload(domain, stock_settings, version, user, last_sync, synclog):
 
 
 def get_case_payload_batched(domain, stock_settings, version, user, last_sync, synclog):
-    response = StringRestoreResponse()
+    response = get_restore_class(user)()
+
+    batch_count = 0
+    sync_operation = BatchedCaseSyncOperation(user, last_sync)
+    for batch in sync_operation.batches():
+        batch_count += 1
+        logger.debug(batch)
+
+        # case blocks
+        case_xml_elements = (
+            xml.get_case_element(op.case, op.required_updates, version)
+            for op in batch.case_updates_to_sync()
+        )
+        response.extend(case_xml_elements)
```

Comment:
that would explain the test error. good catch, thanks. i think this happened when i was merging conflicts